### ISSUE TITLE
suggestion: add k size note to `sig merge` help docs

### ISSUE THREAD

**Issue by jessicalumian** [sourmash-bio/sourmash#2708]
I was running the command 

```
sourmash sig merge *.sig -o ref.sig
```

where each file in *.sig was calculated at k = 21,31,51. So I got the error:

```
== This is sourmash version 4.5.0. ==
== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==

ERROR when merging signature '<file>' (45087801) from file <file>.sig
different ksizes cannot be compared
```

I figured out that I was missing a k flag (`sourmash sig merge *.sig -o ref.sig -k 21`) but the docs for sourmash sig merge don't specifically mention this: https://sourmash.readthedocs.io/en/latest/command-line.html#sourmash-signature-merge-merge-two-or-more-signatures-into-one

`sourmash sig merge -h` does mention this in the output, which is good!

Suggestion: add a note about this in the sourmash sig merge web docs, and potentially to all functions where k size should be specified.

Feel free to ignore this! Just thought I'd mention it. =]

**Comment by ctb** [comment]
Fixed in #2713!

**Comment by ctb** [comment]
@jessicalumian could you confirm that the new docs make sense? see #2713 header for direct links.
